draft-ietf-jsonpath-base.md

```diff
@@ -1272,14 +1272,18 @@ rather than the negated existence test `!@.foo` (which yields false if `@.foo` s
 
 The comparison operators `==` and `<` are defined first and then these are used to define `!=`, `<=`, `>`, and `>=`.
 
-When a path resulting in an empty nodelist appears on either side of a comparison:
+When a path resulting in an empty nodelist or a function expression returning `Nothing` appears on either side of a comparison:
 
 * a comparison using the operator `==` yields true if and only if the comparison
-is between two paths each of which result in an empty nodelist.
+is between:
```

Comment:
maybe just say "the other side also results in an empty nodelist or Nothing"?

Reply:
Very good. I'm including that in a rewording.